setup.cfg

```diff
@@ -1,8 +1,5 @@
-[flake8]
-exclude = .eggs,*.egg,build,docs,.git
-
-[pep8]
-exclude = .eggs,*.egg,build,.git
+[flake8]  # autopep8 would honor this setting, but not the other way around
+exclude = .eggs,*.egg,build,docs,.git,*cupy/core/include/**.py
```

Comment:
This fixed a crazy bug in autopep8 that took me quite a while to understand. The long story in short is autopep8 would pick up both `flake8` and `pep8` sections here, causing the exclude patterns being overwritten in an unpredictable (to me) way. A simpler fix is to duplicate the pattern to both sections to avoid conflicts, but then it's a duplication so we may as well remove one of them 🤦🏻‍♂️

Reply:
This change is reverted in 47314ff since we no longer download the whole git repo of DLPack.